azure-pipelines.yml

```diff
@@ -0,0 +1,48 @@
+# Node.js
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+# if you want continuous build of the master branch, uncomment trigger condition below
+#trigger:
+#- master
+
+pool:
+  vmImage: 'ubuntu-latest'
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
+  displayName: 'Install Node.js'
+
+- script: |
+    npm install
```

Comment:
Think their example uses this `npm install` if it was copied, but I think you want to use `npm ci` here also.